### ISSUE TITLE
Using backticks for `describe` or `it` causes test to not show up

### ISSUE THREAD

**Issue by NoelAbrahams** [connor4312/nodejs-testing#17]
Simple repro:

a) Test bar does not show up.

```ts

describe('foo', () => {

 it(`bar`, () => {
   
   // Assert

  });

});

```

b) foo does not show up

```ts

describe(`foo`, () => {

 it('bar', () => {
   
   // Assert

  });

});

```

**Comment by connor4312** [comment]
Released in 1.3.1

**Comment by NoelAbrahams** [comment]
Works now. Thanks.
